phongo_compat.h

```diff
@@ -58,22 +58,14 @@
 #define PHONGO_BREAK_INTENTIONALLY_MISSING
 #endif
 
-#define phongo_char zend_string
-#define phongo_long zend_long
 #if SIZEOF_ZEND_LONG == 8
 #define PHONGO_LONG_FORMAT PRId64
 #elif SIZEOF_ZEND_LONG == 4
 #define PHONGO_LONG_FORMAT PRId32
 #else
 #error Unsupported architecture (integers are neither 32-bit nor 64-bit)
 #endif
-#define SIZEOF_PHONGO_LONG SIZEOF_ZEND_LONG
-#define phongo_create_object_retval zend_object*
-#define phongo_get_gc_table zval**
 #define PHONGO_ALLOC_OBJECT_T(_obj_t, _class_type) (_obj_t*) ecalloc(1, sizeof(_obj_t) + zend_object_properties_size(_class_type))
-#define PHONGO_TSRMLS_FETCH_FROM_CTX(user_data)
-#define DECLARE_RETURN_VALUE_USED int return_value_used = 1;
```

Comment:
Great catch with this! I did some digging and found that it was introduced in https://github.com/mongodb/mongo-php-driver/commit/23f8316617067f44af59c027768c6470fb53e7f0 for [PHPC-370](https://jira.mongodb.org/browse/PHPC-370). That issue explains that it's meaningless for PHP 7.